back/src/adapters/secondary/notificationGateway/BrevoNotificationGateway.ts

```diff
@@ -59,11 +59,14 @@ export class BrevoNotificationGateway implements NotificationGateway {
   });
 
   constructor(
-    private readonly httpClient: HttpClient<BrevoNotificationGatewayRoutes>,
-    private emailAllowListPredicate: (recipient: string) => boolean,
+    private readonly params: {
+      httpClient: HttpClient<BrevoNotificationGatewayRoutes>;
+      emailAllowListPredicate: (recipient: string) => boolean;
+      defaultSender: RecipientOrSender;
+      blackListedEmailDomains: string[];
+      generateHtmlOptions?: GenerateHtmlOptions;
+    },
```

Comment:
on peut renomer le params là, genre `deps` ou `config` ?
```suggestion
    private readonly config: {
      httpClient: HttpClient<BrevoNotificationGatewayRoutes>;
      emailAllowListPredicate: (recipient: string) => boolean;
      defaultSender: RecipientOrSender;
      blackListedEmailDomains: string[];
      generateHtmlOptions?: GenerateHtmlOptions;
    },
```